### PR TITLE
Refactoring subtree manager RPCs

### PR DESCRIPTION
**Pull request by BranislavMalo**
JIRA: UNIC-1286
TYPE: API

### CODE REVIEW

**Review by knuuzor — COMMENTED** [review]
Copy check OK

**Review by Vasyl-9 — APPROVED** [review]

File: frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_calculate-subtree-git-like-diff/readme.md [hunk 1]
```diff
@@ -99,12 +102,11 @@ curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig
   "errors": {
     "error": [
       {
-        "error-message": "Field target-path is not specified in input request",
+        "error-type": "application",
         "error-tag": "invalid-value",
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_calculate-subtree-git-like-diff/readme.md:106]: probably missing parameter would be better error-tag

File: frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-many/readme.md [hunk 2]
```diff
@@ -78,11 +74,16 @@ curl --location --request POST 'http://localhost:8181/rests/operations/subtree-m
...
 {
-    "output": {
-        "error-message": "Failed to find schema node with identifier '(example-interfaces?revision=2019-12-01)interfaces' under: (example-interfaces?revision=2019-12-01)ext",
-        "result": "fail"
-    }
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "operation-failed",
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-many/readme.md:83]: why 500? it's user's fault

File: frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-one/readme.md [hunk 3]
```diff
@@ -73,11 +60,16 @@ curl --location --request POST 'http://localhost:8181/rests/operations/subtree-m
 }'
 ```
 
-```json RPC Response, Status: 200
+```json RPC Response, Status: 500
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-one/readme.md:63]: same comment

**Review by jaro0149 — CHANGES_REQUESTED** [review]

File: frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_calculate-subtree-git-like-diff/readme.md (hunk 1 above)
Comment (BranislavMalo) [reply frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_calculate-subtree-git-like-diff/readme.md:106]: Fixed in doc. My mistake.

File: frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_bulk-edit/readme.md [hunk 4]
```diff
@@ -212,27 +213,22 @@ curl --location --request POST 'http://127.0.0.1:8181/rests/operations/subtree-m
...
-                "error-message": "Failed to parse path: Could not parse path 'network-topology:network-topology/topology=uniconfig/node=dev02/frinx-uniconfig-topology:configuration/dhcp/profile=test-profile'. Offset: '108': Reason: List entry '(test-ns?revision=2018-09-12)dhcp' requires key or value predicate to be present",
-                "error-type": "uniconfig-error"
-            }
-        ]
-    }
+  "errors": {
+    "error": [
+      {
+        "node-id": "dev01",
+        "error-type": "application"
```
[the remark this answers is not in the document]
Comment (BranislavMalo) [reply frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_bulk-edit/readme.md:224]: Fixed.

File: frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-many/readme.md (hunk 2 above)
Comment (BranislavMalo) [reply frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-many/readme.md:83]: Fixed to unknown-element.

File: frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-one/readme.md (hunk 3 above)
Comment (BranislavMalo) [reply frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_copy-one-to-one/readme.md:63]: Fixed to unknown-element.

**Review by jaro0149 — APPROVED** [review]

**Pull request merged**